addons/payment_adyen/migrations/10.0.1.0/noupdate_changes.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+  <record id="payment.payment_acquirer_adyen" model="payment.acquirer">
```

Comment:
I wouldn't load any of these payment records, as they have been customized for sure.

Reply:
dropped all the payment commits :)